internal/store/proxy_test.go

```diff
@@ -0,0 +1,78 @@
+package store
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	mattermostModel "github.com/mattermost/mattermost-server/v5/model"
+
+	"github.com/mattermost/mattermost-marketplace/internal/model"
+	"github.com/mattermost/mattermost-marketplace/internal/testlib"
+)
+
+func TestProxy(t *testing.T) {
+	t.Run("empty stream", func(t *testing.T) {
+		logger := testlib.MakeLogger(t)
+		ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+			w.WriteHeader(http.StatusOK)
+		}))
+		t.Cleanup(ts.Close)
+
+		proxyStore, err := NewProxy(ts.URL, logger)
+		require.NoError(t, err)
+
+		plugins, err := proxyStore.GetPlugins(&model.PluginFilter{
+			PerPage: model.AllPerPage,
+		})
+		require.NoError(t, err)
+		require.Empty(t, plugins)
+	})
+
+	t.Run("empty stream", func(t *testing.T) {
```

Comment:
`"empty stream with error"` to differentiate from the last test

Reply:
```suggestion
	t.Run("empty stream with error", func(t *testing.T) {
```